genie/utils/support.py

```
@@ -1,42 +1,118 @@
+			headers=headers,
+			payload={
+				"email": user
+			},
+			req_type="GET",
+		)
+
+		if user_data.get("data"):
+			return
+
+	except Exception:
+		pass
```

Ruff (0.15.5)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)